README.md

```diff
@@ -0,0 +1,45 @@
+# Doto
+
+Welcome to Doto. The open-source software (OSS) project for a smart scheduling calendar app. Doto is an online calendar and to-do app. It has all of the basic functionality of any calendar app and can be used to make to-do lists. It also has smart scheduling capabilities, meaning that if the user wants to do a task, it can input it into our app and the app will allocate this task in a suitable time in the user’s calendar. Doto uses Google to sign up, meaning that to use this app, the user must have a google account. The development of Doto is done using the M.E.R.N (MongoDB, Express, React, Node) tech stack (more info about the tech stack can be found in the Wiki).
+
+## Why is this project Useful?
+This project is useful as it has the basic functionalities of any normal calendar app, but also has the added functionality of smart scheduling. This is particularly useful for people who like to plan their day but find it difficult to plan it well, as the app finds the most suitable time for whatever task the user wants to accomplish. It also encourages people who don’t usually like to plan their day to start planning their day as all they need to do is input the task and the app will choose the perfect time, no effort required. 
+
+
+## Development Setup
+It is recommended that you use VsCode when contributing to this project. Please install the Eslint and Prettier extensions so that the code style matches what has already been done before.
+
+## Running the code
+We are using Npm for this particular project (https://www.npmjs.com/get-npm).
+```bash
+Npm install
+```
+```bash
+Npm start 
```

Comment:
it's "Npm" it's "npm" right?

also I would recommend that you write "cd doto-frontend" or "cd doto-backend" or else people will end up creating packages in the root directory.

also "npm run start" not "npm start" right?

Reply:
I checked the front end readme and it says "npm start". Not sure if there is a difference.